src/landing/moudles/components/EmailText.tsx

```diff
@@ -0,0 +1,19 @@
+import TextField from "@material-ui/core/TextField";
+
+const Emailtext = () => {
```

Comment:
Question: if you have already extracted email as a separate component, can we reuse it also in SignUp component ?